fastly/token.go

```diff
@@ -64,17 +64,17 @@ func (c *Client) ListTokens() ([]*Token, error) {
 
 // ListCustomerTokensInput is used as input to the ListCustomerTokens function.
 type ListCustomerTokensInput struct {
-	ID string
+	CustomerID string
```

Comment:
As this is changing functionality, this feels like it should be in a separate PR and not in a test fixtures update PR. As will make documenting in the changelog easier.

Reply:
👍🏻